Cannot find name 'ReactKeycloakContextValue'

**Describe the bug**
Typescript compile error

**To Reproduce**
1. Compile with typescript

**Expected behavior**
No errors happen

**Additional context**
full error message:
`node_modules/@react-keycloak/web/index.d.ts(120,5): error TS2304: Cannot find name 'ReactKeycloakContextValue'.`

using:
@react-keycloak/web: ^1.0.0-alpha.5
keycloak-js: ^8.0.1,
npm: 6.13.4
tsc: 3.7.3

--- a/packages/web/index.d.ts
+++ b/packages/web/index.d.ts
@@ -3,6 +3,7 @@
 // Definitions by: Mattia Panzeri <https://github.com/panz3r>
 // TypeScript Version: 3.4
 import { Component, ComponentType } from 'react'
+import { IReactKeycloakContextProps } from '@react-keycloak/core';
 import {
   KeycloakError,
   KeycloakInitOptions,
@@ -117,7 +118,7 @@ export function withKeycloak<TPromise extends KeycloakPromiseType = 'native'>(
  */
 export type ReactKeycloakHookResult<
   TPromise extends KeycloakPromiseType = 'native'
-> = ReactKeycloakContextValue<TPromise> & [KeycloakInstance<TPromise>, boolean]
+> = IReactKeycloakContextProps<TPromise> & [KeycloakInstance<TPromise>, boolean]
 
 /**
  * Return the Keycloak instance and initialization state.

